app/src/main/java/com/kickstarter/viewmodels/MessageHolderViewModel.java

```diff
@@ -0,0 +1,119 @@
+package com.kickstarter.viewmodels;
+
+import android.support.annotation.NonNull;
+import android.support.v4.util.Pair;
+
+import com.kickstarter.R;
+import com.kickstarter.libs.ActivityViewModel;
+import com.kickstarter.libs.CurrentUserType;
+import com.kickstarter.libs.Environment;
+import com.kickstarter.models.Message;
+import com.kickstarter.models.User;
+import com.kickstarter.services.ApiClientType;
+import com.kickstarter.ui.viewholders.MessageViewHolder;
+
+import rx.Observable;
+import rx.subjects.PublishSubject;
+
+import static com.kickstarter.libs.rx.transformers.Transformers.combineLatestPair;
+import static com.kickstarter.libs.rx.transformers.Transformers.neverError;
+
+public interface MessageHolderViewModel {
+
+  interface Inputs {
+    /** Call to configure the view model with a message. */
+    void configureWith(Message message);
+  }
+
+  interface Outputs {
+    /** Emits a boolean that determines whether the creator's avatar image should be hidden. */
+    Observable<Boolean> creatorAvatarImageHidden();
```

Comment:
i think this output should be something like `participantAvatarImageHidden`, since the other person in the convo won't necessarily be the creator (like in the future when we have creator tools).

same for the other creator output...